packages/integration/lib/src/widgets/user/user_created_comment_listview.dart

```diff
@@ -309,6 +313,218 @@ class _LMFeedUserCreatedCommentListViewState
     );
   }
 
+  Widget _defPollWidget(LMFeedPoll pollWidget, LMPostViewData postViewData) {
```

Comment:
Return type

Reply:
here return type will be `Widget` since it is origially called in `LMFeedPostMedia` where the return type is `LMFeedPollWidget` on feed_screen and other screens we are just ovverriding them with it's implementation